modules/rosa-policy-incident.adoc

```diff
@@ -41,49 +41,46 @@ The following activities can trigger notifications:
 - Upgrade scheduling
 
 [id="rosa-policy-backup-recovery-sts_{context}"]
-== Backup and recovery for ROSA clusters with STS
+== Infrastructure and data resiliency
+Customers are responsible for taking regular backups of their data and should deploy multi-AZ clusters with workloads that follow Kubernetes best practices to ensure high availability within a region. If an entire cloud region is unavailable, customers must install a new cluster in a different region and restore their apps using their backup data.
 
-There is no backup method available for ROSA clusters with STS.
+There is no backup method available for ROSA clusters with STS. Red Hat does not commit to any Recovery Point Objective (RPO) or Recovery Time Objective (RTO).
```

Comment:
```suggestion
There is no Red Hat provided backup method available for ROSA clusters with STS. Red Hat does not commit to any Recovery Point Objective (RPO) or Recovery Time Objective (RTO).
```

Could like to https://docs.openshift.com/rosa/rosa_backing_up_and_restoring_applications/backing-up-applications.html#oadp-installing-oadp-rosa-sts_rosa-backing-up-applications ?